data/templates/ssh/sshd_config.j2

```diff
@@ -30,6 +29,7 @@ PermitRootLogin no
 PidFile /run/sshd/sshd.pid
 AddressFamily any
 DebianBanner no
+PasswordAuthentication no
```

Comment:
This looks like it'd cause login issues for passwords without 2FA?

Reply:
PasswordAuthentication replaced with ChallengeResponseAuthentication. As recommended in the official OpenSSH documentation
https://www.ssh.com/academy/ssh/sshd_config
Also, I did a test, password only authentication works with these changes.